docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -33,7 +34,7 @@ The authorization code flow is completed and the Quarkus web-app uses the tokens
 The following tokens are issued:
 
 * ID token: The Quarkus web-app uses the user information in the ID token to enable the authenticated user to log in securely and to provide role-based access to the web-app.
```

Comment:
@michelle-purcell Lets keep it as `web-app` to avoid the confusion as it is an application type requiring a code flow authentication. `web app` is a general concept, we can have `web app` protected by the bearer token authentication or any other security mechanism

Reply:
Also propose to have `application` instead of `component`